docs/concepts/experiment/sessions.md

```diff
@@ -0,0 +1,28 @@
+# Experiment Sessions
+
+### What are sessions?
+Think of a session as a conversation between two people. A session includes all messages exchanged during a conversation. A new message from the participant is interpreted in the context of the current conversation (or session). OCS achieves this by including all previous messages from a particular session in the system prompt as conversation history. This system prompt, along with the participant's message, is sent to the LLM to generate context-appropriate responses.
```

Comment:
It's not quite true that we put the history into the system prompt. The history is sent as separate messages in the API.

Reply:
This whole time I thought for some reason that we included it in the system message. Huh..